Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/hostedtoolcache/Python/3.8.16/x64/lib/python3.8/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
np_2d = np.array([1.73, 1.68, 1.71, 1.89, 1.79],
                 [65.4, 59.2, 63.6, 88.4, 68.7])
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mTypeError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[26], line 1[0m
[0;32m----> 1[0m np_2d [38;5;241m=[39m [43mnp[49m[38;5;241;43m.[39;49m[43marray[49m[43m([49m[43m[[49m[38;5;241;43m1.73[39;49m[43m,[49m[43m [49m[38;5;241;43m1.68[39;49m[43m,[49m[43m [49m[38;5;241;43m1.71[39;49m[43m,[49m[43m [49m[38;5;241;43m1.89[39;49m[43m,[49m[43m [49m[38;5;241;43m1.79[39;49m[43m][49m[43m,[49m
[1;32m      2[0m [43m                 [49m[43m[[49m[38;5;241;43m65.4[39;49m[43m,[49m[43m [49m[38;5;241;43m59.2[39;49m[43m,[49m[43m [49m[38;5;241;43m63.6[39;49m[43m,[49m[43m [49m[38;5;241;43m88.4[39;49m[43m,[49m[43m [49m[38;5;241;43m68.7[39;49m[43m][49m[43m)[49m

[0;31mTypeError[0m: Field elements must be 2- or 3-tuples, got '65.4'
TypeError: Field elements must be 2- or 3-tuples, got '65.4'

